src/avram/migrator/columns/primary_keys/base.cr

```diff
@@ -0,0 +1,11 @@
+abstract class Avram::Migrator::Columns::PrimaryKeys::Base
+  macro inherited
+    private getter name : String
+  end
+
+  abstract def column_type
+
+  def build : String
+    %(  #{name} #{column_type} PRIMARY KEY)
```

Comment:
What are your thoughts on adding an extra method for default value here?

```crystal
%(  #{name} #{column_type} PRIMARY KEY #{default_values})
```

In the case of Int32 and Int64 being serial and bigserial, their methods would define nothing. But in the case of UUID, it would define 

```crystal
def default_values
  "DEFAULT uuid_generate_v4()"
end
```

One thing I'm also thinking about is later in another PR may be the ability to just define a String as primary key, or possibly composite keys. We will need a way to tell it where it gets it's default value from.

Reply:
I think in those cases they can override the `build` method so they can do whatever they want. But I agree it should do this by default for uuid. I added an issue here https://github.com/luckyframework/avram/issues/122 to tackle that.